Document/0x05b-Basic-Security_Testing.md

```diff
@@ -153,11 +153,10 @@ See also test case "Testing Custom Certificate Stores and SSL Pinning" for furth
 
 ##### Root Detection
 
-To implement root detection on Android, libraries like RootBeer<sup>[14]</sup> or custom checks are used to verify if the device is rooted or not. See also test case "Testing Root Detection" and "Testing Advanced Root Detection" for further details.
+To implement root detection on Android, libraries like RootBeer<sup>[14]</sup> or custom checks are used to verify if the device is rooted or not. Please note that root detection mechanisms are relatively easy to circumvent and should only be used to support, but not base the app security upon. See also test case "Testing Root Detection" and "Testing Advanced Root Detection" for further details.
```

Comment:
> To implement root detection on Android, libraries like RootBeer<sup>[14]</sup> or custom checks are used to verify if the device is rooted or not.

This sentence contains two times a subsentence with "to": "To implement root detection on Android", "to verify if the device is rooted or not". This makes it a bit awkward, and it still does not explain why you would want root protection. Maybe you can change this sentence to make it match your added sentence better.

> Please note that

I am not sure this properly indicates the relation to the rest of the paragraph.

> root detection mechanisms are relatively easy to circumvent

Maybe you can provide a pointer about how the mechanisms can be bypassed?

> be used to support, but not base the app security upon.

I think this part of the sentence doesn't really work, because in the first part of the sentence it is not clear what should be supported, and the "upon" messes that up. 

I think the fact that root detection is easy to bypass is useful information for a mobile pentester, but I am not sure whether it belongs in this chapter. This chapter is more about getting an app working on a rooted device so that you can test it, and not about the design decisions when creating a mobile app. Even so, this chapter could use some pointers about how to bypass the root detection, or at least a note that it is possible.